dsms/core/configuration.py

```diff
@@ -58,6 +58,13 @@ class Configuration(BaseSettings):
         description="Repository of the triplestore for KItems in the DSMS",
     )
 
+    individual_slugs: bool = Field(
```

Comment:
Perhaps `automatic_slugs` or `autocomplete_slug` would be a more self-explantory name.